src/doc/rustc/src/targets/custom.md

```diff
@@ -15,3 +15,14 @@ rustc +nightly -Z unstable-options --target=wasm32-unknown-unknown --print targe
 ```
 
 To use a custom target, see the (unstable) [`build-std` feature](../../cargo/reference/unstable.html#build-std) of `cargo`.
+
+## Custom Target Lookup Path
+
+When `rustc` is given an option `--target=TARGET` (where `TARGET` is any string), it uses the following logic:
+1. if `TARGET` is the name of a built-in target, use that
+2. if `TARGET` is a path to a file, read that file as a json target
+3. otherwise, search the colon-seperated list of directories found
+   in the `RUST_TARGET_PATH` environment variable for a file named
+   `TARGET.json`.
```

Comment:
Nit: this should also explicitly say the "first" found `TARGET.json` right?

Reply:
hmm, grammatically, i don't think so, but i could clarify that it's searched in order, even though the similarity to `PATH` will imply that to a lot of programmers

Reply:
> hmm, grammatically, i don't think so, but i could clarify that it's searched in order, even though the similarity to `PATH` will imply that to a lot of programmers

Right. In this case, since we're documenting behavior, I think we want to be explicit and don't rely on assumptions.

Reply:
yeah, it's better to be explicit.  i'll fix it as soon as i'm back at my laptop.